include/camp/tuple.hpp

```diff
@@ -124,23 +124,41 @@ namespace internal
   template <typename... Types, camp::idx_t... Indices>
   struct tuple_helper<camp::idx_seq<Indices...>, camp::list<Types...>>
       : public internal::tuple_storage<Indices, Types>... {
-    template <bool B = concepts::metalib::all_of<
-                  std::is_default_constructible<Types>::value...>::value,
-              typename std::enable_if<B, void>::type* = nullptr>
+
+#if (!defined(__NVCC__)) \
+        || (__CUDACC_VER_MAJOR__ > 10 \
+          || (__CUDACC_VER_MAJOR == 10 && __CUDACC_VER_MINOR__ >= 1))
```

Comment:
@rhornung67 @trws The error is on this line. There is a missing "_ _" suffix to "__CUDACC_VER_MAJOR". Tested this with nvcc10+gcc7.3.1 and clang8.0.1. Have not tried nvcc9 yet, and XL is still building.

Reply:
Good catch! 🥇 

So that fixes the build error I saw, correct?  Are the camp warnings gone too?

Reply:
Warnings are gone too (apart from a couple of the TypedIndex warnings we had before, but those are few and far between).

Reply:
Excellent. We can probably do a RAJA release next week if @trws is good with these changes.  I'll start some testing with this against the perf suite and get a RAJA RC branch started.

Reply:
NVCC9 still has the same build error. It fails for the non-default version of tuple_helper& operator=(). I'll look at this some more.

Reply:
I'm all good with that change, not sure what the nvcc9 issue is about, the camp tests pass so it must be some interaction issue, will try to track it down.